web/src/models/enums/Genders.ts

```diff
@@ -0,0 +1,6 @@
+export enum Genders {
+    Woman,
+    Man,
+    NonBinary,
```

Comment:
the fuck is non binary. there is no non binary its either 1 or 0 there is no middle. if he got pipi he man if she got vajina he womeb

Reply:
AHXSDJAXDSGXASDH

Reply:
Admini nonbinary yaptım